pkg/runtime-tools/generate/generate.go

```diff
@@ -325,6 +337,23 @@ func (g *Generator) AdjustDevices(devices []*nri.LinuxDevice) {
 	}
 }
 
+// InjectCDIDevices injects the given CDI devices into the OCI Spec.
+// Devices are given by their fully qualified CDI device names. The
```

Comment:
Could we use `requested` or `specified` instead of `given`?

Reply:
Updated, using requested instead.

Reply:
```suggestion
// Devices are requested by their fully qualified CDI device names. The
```
(this could also be `specified`).